dreammall-earth/dreammall.earth
# fix(other): scale down+up one controle plane node

Motivation
----------
So, I wasn't able to SSH into the control plane node in Nuremberg data center anymore and the node wouldn't update to K3S version `v1.29.9+k3s1` anymore.

I just scaled up + down the `count` like from `1` to -> `0` and back to -> `1`. In between I would wait for the pods to recover.

Now I can see that the respective node `kube-dreammall-control-plane-nbg1-mgo` has updated the K3s version and is accessible via SSH. Good! However it's displayed as `Ready,SchedulingDisabled` (before `NotReady,SchedulingDisabled`). I hope it's the expected state for 3 control planes.

![image](https://github.com/user-attachments/assets/ef2d4122-d75a-4ccd-977d-b00e59228cee)

How to test
-----------
1. You could check `k9s` by yourself. Type `:nodes` to see the state of nodes.
